packages/relay/tests/lib/hbarLimiter.spec.ts

```diff
@@ -207,15 +211,40 @@ describe('HBAR Rate Limiter', async function () {
   });
 
   it('Should preemtively limit while expected transactionFee is greater than remaining balance', () => {
-    const validTotalTxFee = validTotal - 100;
-    const invalidTotalTxFee = validTotal + 100;
-    rateLimiter = new HbarLimit(logger, currentDateNow, validTotal, validDuration, registry);
+    rateLimiter = new HbarLimit(logger, currentDateNow, invalidTotal, validDuration, registry);
```

Comment:
Can we avoid instantiating this inside every test and move it to `beforeAll`? 

We can declare a separate object in the tests where we want this to be reinstantiated (e.g. after changing an env variable), but for the tests which don't do anything additional can reuse the one that was instantiated in the `beforeAll` method instead of creating a new one for every test.

Reply:
I have a different perspective on this. I'm in favor of reinstantiating the `rateLimiter` object so that each unit test starts with a fresh instance, which, I guess, the OA of these tests had in mind. Well these tests are designed to modify attributes within the `rateLimiter` class like the expenses, remaining balances, etc.. Reusing the same object would carry over the updated values from one test to another, causing them to depend on each other and undermining the principle that unit tests should be independent and not influenced by changes from other tests.

Pushed in an update to clean up the test suite and add the rateLimitter instantiation to `beforeEach`, instead of `beforeAll`.